### DISCUSSION TITLE
Unable to specify individual 'ax_scale' for wrap, row and col plots/figures

### DISCUSSION THREAD

**Discussion by spaderdabomb** [endangeredoxen/fivecentplots#52] [Ideas]
Is there a way to individually address 'ax_scale' when doing wrap, row or col plots/figures? I am trying to make a figure with multiple plots but the y-axis for each plot need different y-scales (linear & log).

**Comment by endangeredoxen** [comment]
It looks like I never implemented this, but it is a worthwhile feature to add.  I'll put it in the queue and thanks for the suggestion

**Reply by spaderdabomb** [reply]
Cool! Hope to see this feature in the future. Very awesome and useful package =).
